### ISSUE TITLE
It rather involved being on the other side of this airtight hatchway

### ISSUE THREAD

**Issue by parsiya** [parsiya/parsiya.net#19]
Talk about why "You need to be admin to get admin" is not useful.

Evergreen blog post by Raymond Chen: https://devblogs.microsoft.com/oldnewthing/20060508-22/?p=31283
"It rather involved being on the other side of this airtight hatchway"

It rather involved being on the other side of this airtight hatchway: If they can run code, then they can run code:
https://devblogs.microsoft.com/oldnewthing/20070706-00/?p=26123

It rather involved being on the other side of this airtight hatchway: If they can inject code, then they can run code - different from above: https://devblogs.microsoft.com/oldnewthing/20100114-00/?p=15273

Being able to call a function without using GetProcAddress is not a security vulnerability: https://devblogs.microsoft.com/oldnewthing/?p=18593

And another, this is the link to "found a complicated way of doing something perfectly mundane" in the previous blog post, "If users can shut down the machine, it's not a security hole if they can shut down the machine": https://devblogs.microsoft.com/oldnewthing/?p=22293

Follow up after 12 years, this directly talks about DLL hijacking where standard user does not have write access:

It rather involved being on the other side of this airtight hatchway: Replacing an unsigned writable MSI package: https://devblogs.microsoft.com/oldnewthing/20180227-00/?p=98115

I like this quote a lot

"In other words, you just found a complicated way of doing something perfectly mundane. You can’t make up for the absence of any actual vulnerability by piling on style points and cranking up the degree of difficulty."

Reporting on what you could do once you get to the other side of the airtight hatchway: https://devblogs.microsoft.com/oldnewthing/20200518-00/?p=103767

### DLL Hijacking in SYSTEM32.
Blaming the operating system for allowing people to create files with unusual characters in their names: https://devblogs.microsoft.com/oldnewthing/20200617-00/?p=103870

Tons more. Can't really cover them all.

* https://devblogs.microsoft.com/oldnewthing/?s=airtight+hatchway&submit=%EE%9C%A1

Actual conversation from hitchhiker's guide to the galaxy:

```
Arthur: But can't you think of something?!
Ford: I did.
Arthur: You did!
Ford: Unfortunately, it rather involved being on the other side of this airtight hatchway—
Arthur: oh.
Ford: —that's just sealed behind us.
—HHGTTG, fit the second
```

**Comment by parsiya** [comment]
Find the rest of the blogs, too. Seems like I have missed some of them in the list above.

**Comment by parsiya** [comment]
Closed in c884ffddaadb52b3c2626d99dc915adbc0b577ea
